unix/w0vncserver/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+add_executable(w0vncserver
+  w0vncserver.cxx
+  RFBTimerSource.cxx
+  WSocketMonitor.cxx
+  WDesktop.cxx
+  WPixelBuffer.cxx
+  portals/Screencast.cxx
+  portals/Portal.cxx
+  portals/RemoteDesktop.cxx
+  Pipewire.cxx
+  PortalBackend.cxx
+  ../../common/rfb/PixelBuffer.cxx
+)
+
+find_package(PkgConfig REQUIRED)
```

Comment:
Please keep dependencies in the top `CMakeLists.txt` so the requirements of the project are easily seen.